sys-utils/nsenter.c

```diff
@@ -189,6 +193,40 @@ static void open_namespace_fd(int nstype, const char *path)
 	assert(nsfile->nstype);
 }
 
+static void open_target_sk_netns(void)
+{
```

Comment:
It would be nice to avoid global variables there. For example, sock_fd and target_pid could be arguments of the function.